support-frontend/assets/components/sepaForm/SepaForm.tsx

```diff
@@ -0,0 +1,120 @@
+import { css } from '@emotion/react';
+import type { Country } from '@guardian/consent-management-platform/dist/types/countries';
+import { headline, space } from '@guardian/source-foundations';
+import {
+	Option as OptionForSelect,
+	Select,
+	TextInput,
+} from '@guardian/source-react-components';
+import { countries } from 'helpers/internationalisation/country';
+import { sortedOptions } from '../forms/customFields/sortedOptions';
+
+// -- Styles -- //
+const containerStyles = css`
+	padding-top: ${space[5]}px;
+`;
+const headerStyles = css`
+	${headline.xxxsmall({
+		fontWeight: 'bold',
+	})}
+`;
+const fieldsContainerStyles = css`
+	margin-top: ${space[4]}px;
+	> * + * {
+		margin-top: ${space[3]}px;
+	}
```

Comment:
I'm not wild about this kind of complex selector. If we just need things in a column with regular space in between we can use the [Stack](https://guardian.github.io/source/?path=/docs/packages-source-react-components-stack--playground) component from Source.

Reply:
Done!